turbopack/crates/turbo-tasks-backend/src/lmdb_backing_storage.rs

```diff
@@ -249,14 +249,6 @@ impl BackingStorage for LmdbBackingStorage {
         meta_updates: Vec<ChunkedVec<CachedDataUpdate>>,
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line -243]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Maybe also include lengths of the other fields in tracing?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/vercel/next.js/71092?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->